.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Mind adding a comment about why Windows is disabling, linking back to the actions issue? Then we won't scratch our heads in a year or two.

Reply:
b07076b